### ISSUE TITLE
equirectangular images on mobile

### ISSUE THREAD

**Issue by bniesz** [mpetroff/pannellum#64]
Equirectangular images are rendering incorretly on mobile devices - including in the provided examples.  Specifically iOS 8.2.  It almost looks as if they're being rendered as shearicle images instead.

Cube images work ok everywhere.  Equirectangular images work ok on desktop.

Any quick fix for this?

**Comment by bniesz** [comment]
After further experimentation, it appears that there is a pixel dimention limit issue. Lowering the resolution of the equirectangular image seems to resolve.

**Comment by mpetroff** [comment]
Which version of Pannellum are you using, the last release or the Git master?

**Comment by ewirt** [comment]
I encountered this today playing with Pannellum for the first time.  I was originally using the latest release (2.1.1) but then downloaded and tried the Git master, which had the same issue.  I did a little testing and found that a 3238x1619 equirectangular jpeg works fine, but a 3239x1620 image does not.  I tested with iOS 8.2 on an iPad Mini, and iOS 8.3 on an iPhone 5S with the same results on both.  The working image was 2,459,483 bytes in size, and the not-working one was 2,460,502 bytes.  I haven't dug any more to see if I can determine what is causing it to flip out, but figured this might help give you a starting place if you have an opportunity to look into it.
